pkg/provider/aws/resources/lambda.go

```diff
@@ -84,13 +84,29 @@ func (lambda *LambdaFunction) Create(dag *core.ResourceGraph, params LambdaCreat
 }
 
 type LambdaFunctionConfigureParams struct {
-	ConnectionPoolConfigurationInfo ConnectionPoolConfigurationInfo
+	Timeout              int
+	MemorySize           int
+	EnvironmentVariables core.EnvironmentVariables
 }
 
 // Configure sets the intristic characteristics of a vpc based on parameters passed in
 func (lambda *LambdaFunction) Configure(params LambdaFunctionConfigureParams) error {
 	lambda.Timeout = 180
```

Comment:
How do we propagate these defaults back into config so that the one we write in the compiled directory is accurate?

Reply:
well my thought is that the ones in the compiled directory are what are setting these unless they are completely overwritten by the specific exec unit or default. In doing so it requires us to create these by doing constraints on the nodes configuration itself which is still something we will have to figure out